docs/content/Cube.js-Frontend/@cubejs-client-vue.md

```diff
@@ -145,8 +159,7 @@ chartType? | [ChartType](#types-chart-type) | The type of chart to display. |
 
 #### Empty Slot
 
-This slot functions as a empty/loading state in which if the query is loading or empty so you can show
-something in the meantime.
+This slot functions as a empty/loading state in which if the query is loading or empty so you can show something in the meantime.
```

Comment:
Do we want to make this consistent with line 36 above? I personally like the `;` as it reads better.